cwf/k8s/cwf_operator/deploy/operator.yaml

```diff
@@ -58,3 +58,6 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "cwf-operator"
+            # Replace value for redis server address to enable status reporting
+            # - name: REDIS_ADDR
+            #  value: localhost:6380
```

Comment:
?

Reply:
The redis-server address will be based on the redis helm deployment so we default configuration doesn't really work